src/main/java/org/jenkinsci/plugins/workflow/support/steps/build/RunWrapper.java

```diff
@@ -162,6 +162,11 @@ public String getFullProjectName() throws AbortException {
     }
 
     @Whitelisted
+    public List<String> getLog(int maxLines) throws IOException {
+        return build().getLog(maxLines);
```

Comment:
Possibly bad for performance if the argument is large. On the other hand sometimes you would want the full log. https://github.com/jenkinsci/workflow-basic-steps-plugin/pull/2 might provide a more scalable solution, at least for the common case of scanning for messages in the current build’s log after an error (this method could be used for other purposes, such as after a `build` step).